pentaho/src/test/java/pt/webdetails/cpf/olap/AbstractOlapUtilsTest.java

```diff
@@ -78,12 +80,10 @@ public void testGetOlapCubes() throws Exception {
         "\"jndi\":\"testJndi\"},{\"schema\":\"catalog2Definition\",\"name\":\"catalog2Name\"," +
         "\"cubes\":[{\"id\":\"identifier0\",\"name\":\"name0\"},{\"id\":\"identifier1\",\"name\":\"name1\"}]," +
         "\"jndi\":\"testJndi\"}]}";
-    try {
-      result = olapUtils.getOlapCubes();
-    } catch ( JSONException e ) {
-      Assert.fail( e.getMessage() );
-    }
-    Assert.assertTrue( jsonEquals( expectedResult, result.toString() ) );
+
+    result = olapUtils.getOlapCubes();
+
+    assertTrue( jsonEquals( expectedResult, result.toString() ) );
```

Comment:
Can you use here the assertEquals?

Reply:
No, because the comparison involves some intermediate transformations.